src/amuse/community/galactics/gas_interface.py

```diff
@@ -764,11 +764,11 @@ def update_particle_set(self):
         number_of_updated_particles,number_of_updated_gas_particles = self.get_number_of_particles_updated()
         if number_of_updated_particles:
             self.particles._private.attribute_storage._add_indices(
-                range(number_of_updated_gas_particles,number_of_updated_particles)
+                list(range(number_of_updated_gas_particles,number_of_updated_particles))
```

Comment:
this is a recurring pattern: functions should be able to handle iterators, and then the list here can be removed...

  we should generate an issue for this

Reply:
yep... Not sure how to formulate it yet...

Reply:
Issue #488